internal/app/certificate_provider_store.go

```diff
@@ -49,14 +56,40 @@ func (s *certificateProviderStore) Create(ctx context.Context, donorSessionID st
 	return cp, err
 }
 
-func (s *certificateProviderStore) GetAny(ctx context.Context) (*actor.CertificateProviderProvidedDetails, error) {
+func (s *CertificateProviderStore) CreatePaper(ctx context.Context, lpaID string, certificateProviderUID actoruid.UID, donorSessionID string) error {
+	cp := &actor.CertificateProviderProvidedDetails{
+		PK:        lpaKey(lpaID),
+		SK:        certificateProviderKey(certificateProviderUID.String()),
+		UID:       certificateProviderUID,
+		LpaID:     lpaID,
+		UpdatedAt: s.now(),
+		Certificate: actor.Certificate{
+			AgreeToStatement: true,
+			Agreed:           s.now(),
+		},
+	}
+
+	if err := s.dynamoClient.Create(ctx, cp); err != nil {
+		return err
+	}
+
+	return s.dynamoClient.Create(ctx, lpaLink{
+		PK:        lpaKey(lpaID),
+		SK:        subKey(certificateProviderUID.String()),
```

Comment:
So this obviously isn't a GOL sub but this felt preferable than having to handle two different prefixes for CP SKs. When we come to look at paper CPs moving online we may want to revisit this but for now it means our existing CP functionality for progress and pay/sign logic continues to work.